chart/ory/charts/gcloud-sqlproxy/README.md

```diff
@@ -0,0 +1,145 @@
+
+# GCP SQL Proxy
+
+[sql-proxy](https://cloud.google.com/sql/docs/postgres/sql-proxy) The Cloud SQL Proxy provides secure access to your Cloud SQL Postgres/MySQL instances without having to allow IP addresses or configure SSL.
+
+Accessing your Cloud SQL instance using the Cloud SQL Proxy offers these advantages:
+
+* Secure connections: The proxy automatically encrypts traffic to and from the database; SSL certificates are used to verify client and server identities.
+* Easier connection management: The proxy handles authentication with Google Cloud SQL, removing the need to provide static IP addresses of your GKE/GCE Kubernetes nodes.
+
+## Introduction
+
+This chart creates a Google Cloud SQL proxy Deployment and Service on a Kubernetes cluster using the Helm package manager.
+You need to enable Cloud SQL Administration API and create a service account for the proxy as per these [instructions](https://cloud.google.com/sql/docs/postgres/connect-container-engine).
```

Comment:
as per these instructions > according to the instructions at [Google Cloud documentation](https://cloud.google.com/sql/docs/postgres/connect-container-engine).